examples/basic.js

```diff
@@ -1,6 +1,39 @@
-var feathers = require('feathers'),
-  bodyParser = require('body-parser'),
-  nedbService = require('../lib');
+// server.js
```

Comment:
We should rename this file to `app.js` to be consistent with the other adapters.